templates/astropy.tpl

```diff
@@ -26,7 +26,47 @@
     </nav>
 
     {{ super() }}
+	
+	<!-- Author and Date Information -->
```

Comment:
I suggest this be at the *top* (i.e. above the name) instead of at the bottom.  As it is right now it's easy to miss.